### PR TITLE
Feature/comments

### PR DESCRIPTION
**Pull request by Eunicegenel**
- It is now possible to edit and delete a previously posted comment
- Only the respective users can edit or delete a comment they posted
- Once a user is signed in, the container field for the products will be moved 15rem to the right

homepage without user being logged in
image

homepage with logged in user
image

action button for update and delete is only visible to the user who posted the comment
image

action button options upon click
image

update form appears after clicking edit post
image

updated comment upon changing the text, then clicking update post
image

comment is deleted upon clicking delete post in options
image

### CODE REVIEW

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -37,9 +37,35 @@
...
+              <h5 class="commenter_description"><%= t.body %></h5>
+            </div>
+            <% if current_user.id === t.commenter_id %>
+              <%= javascript_pack_tag 'comment.js' %>
+              <div class="dropdown">
+                <button class="dropdown_btn" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  <%= image_tag("options.png", :class => "option") %>
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                  <a class="dropdown-item edit" id=<%= t.id %>>Edit Post</a>
```
Comment (villanuevajamesfvillanueva) [inline app/views/items/show.html.erb]: This should be edit comment instead

File: app/views/items/show.html.erb [hunk 2]
```diff
@@ -37,9 +37,35 @@
...
+                  <%= form_with scope: :comment, method: :delete, url: delete_comment_path(t.id), local: true do |f| %>
+                    <%= f.submit :"Delete Post", :class => "dropdown-item" %>
```
Comment (villanuevajamesfvillanueva) [inline app/views/items/show.html.erb]: This should be delete comment instead

**Review by villanuevajamesfvillanueva — CHANGES_REQUESTED** [review]

File: app/views/items/show.html.erb (hunk 1 above)
Comment (Eunicegenel) [reply app/views/items/show.html.erb]: instead of edit post? okay, will change

File: app/views/items/show.html.erb (hunk 2 above)
Comment (Eunicegenel) [reply app/views/items/show.html.erb]: will change this as  well

**Review by leocsdev — APPROVED** [review]

**Review by villanuevajamesfvillanueva — APPROVED** [review]

**Pull request merged**